doc/user_guide.dox

```diff
@@ -0,0 +1,207 @@
+`apt` (`apt install virtualgl`). For other client systems refer to the
+official <a href="https://virtualgl.org">VirtualGL documentation</a>for
+installation instructions.
+
+#### Blue Brain V (BB5)
+
+To log into this cluster the recommended method is to use `vizsh`, this will
+setup the necessary environment for you. You can check the
+<a href="https://bbpteam.epfl.ch/project/spaces/display/INFRA/Analysis+and+Visualization+Cluster">`vizsh` documentation</a> for details. All BBP desktop
+systems handled by puppet have `vizsh` already installed, otherwise contact
+the system admins. For basic purposes, the login command is <kbd>vizsh \-\-account=<var>projID</var></kbd>. If you need exclusive access to one node use
+<code>vizsh -n 1</code>. Allocations will take place on UC2 nodes.
+
+If `vizsh` does not work for you or is not available in your machine, you can
+also connect to a front-end node and allocate an interactive job where RTNeuron
+can be run. The instructions to do it are:
+
+\verbatim
+$ ssh bbpv1 # or bbpv2
+you@bbpv1$ salloc --constrain=uc2 --gres=gpu:1 -p interactive --account=projID
+you@rxyz$ # The node varies
+
+Then, from another shell in the client machine
+$ vglconnect node_xyz
+you@rxyz$
+This is the shell from which RTNeuron must be launched
+
+\endverbatim
+
+Once logged, RTNeuron can be accessed using nix shell commands or as a GNU
+environmental module. For simplicity, only the latter method is described.
+To load RTNeuron into your environment type:
+
+\verbatim
+module purge
+module load nix/viz/rtneuron
+\endverbatim
+
+#### Galileo
+
+The recommended way of accessing the system is through <a href=https://wiki.u-gov.it/confluence/pages/viewpage.action?pageId=68391634>RCM</a>, a tool provided
+by Cineca to establish VNC connections to their systems. Please consult the
+site administrators for details about using this tool. Once logged in, type the
+following commands to bring RTNeuron into your shell environment:
+
+\verbatim
+module use /gpfs/work/HBP_CDP21_it_1/HBP/galileo/modules/nix/share/modulefiles
+module purge
+module load nix/viz/rtneuron
+\endverbatim
+
+The load command will print some warnings that can be safely ignored.
+
+#### Jureca
+
+The recommended way of accessing the system is through <a hread=https://trac.version.fz-juelich.de/vis/wiki/vnc3d/strudel>strudel</a>, a tool
+provided by Julich to establish VNC connections to their systems. Please
+consult the site administrators for details about using this tool. Once
+logged in, type the following commands to bring RTNeuron into your shell
+environment:
+
+\verbatim
+module use /work/vsk25/vsk2518/HBP/jurecavis/modules/nix/share/modulefiles;
+export LD_LIBRARY_PATH=/usr/lib64/nvidia:$LD_LIBRARY_PATH
+module purge
+module load nix/viz/rtneuron
+\endverbatim
+
+
+### Once RTNeuron is installed/ready to use
+
+From a shell console run `rtneuron` or `vglrun rtneuron` if you are remotely
```

Comment:
The monospace text is bad to see. Maybe enclose it with quotes (')?

Reply:
I prefer to keep `  ` in the sources because doxygen converts that into <code> </code> tags which can be easily tweaked with the CSS file. If you're more specific about what do you mean with "bad to see" (lack of contrast with the regular text?) I can address it in this review.

Reply:
![selection_454](https://user-images.githubusercontent.com/12729291/46341217-e149a100-c637-11e8-9347-9184ed4ff5c4.png)

In the browser (firefox) the text renders so that I cannot easily distinguish the normal text and the commands to run...

Reply:
Using a different color (dark gray) would be sufficient for you? Or do you want to have a different typeface or font?